Formula/node_exporter.rb

```diff
@@ -0,0 +1,64 @@
+class NodeExporter < Formula
+  desc "Prometheus exporter for machine metrics"
+  homepage "https://prometheus.io/"
+  url "https://github.com/prometheus/node_exporter/archive/v0.16.0.tar.gz"
+  sha256 "2ed1c1c199e047b1524b49a6662d5969936e81520d6613b8b68cc3effda450cf"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    mkpath buildpath/"src/github.com/prometheus"
+    ln_sf buildpath, buildpath/"src/github.com/prometheus/node_exporter"
+
+    system "make", "build"
```

Comment:
This seems to be using `go get`, which we don't allow.

Reply:
Indeed it does. Plain `go build` fortunately does work...